commit c092e10a2d7d8555aba19e96e7716105ff25585f (refs/changes/27/4827/1)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-09-12T15:46:43+02:00 (1 year, 1 month ago)
    
    TRITON-739 Get rid of "$SERVICE_root_pw" and other deprecated values for usb-headnode config

diff --git a/docs/config.md b/docs/config.md
index 76c3d04..a95c752 100644
--- a/docs/config.md
+++ b/docs/config.md
@@ -11,7 +11,7 @@ apisections: SDC Config
 -->
 
 <!--
-    Copyright (c) 2014, Joyent, Inc.
+    Copyright (c) 2018, Joyent, Inc.
 -->
 
 # SDC Config
@@ -52,7 +52,6 @@ configure Headnode global zone:
 - `utc_offset` [^1]: String containing the numeric value for the headnode UTC
   offset, (default `"0"`).
 - `agents_root` [^1]: Absolute path to agents directory.
-- `zonetracker_database_path` [^1]: Absolute path to zonetracker SQLite DB file.
 - `root_authorized_keys_file`[^1]: The name of the file to be used as SSH
   authorized keys source for `root` account.
 - `coal`: Is the current setup a COAL setup?
@@ -78,13 +77,6 @@ The URL to upload service bundles, with the required user name and password.
 Usually, these variables shouldn't be modified at all: `sbapi_url`,
 `sbapi_http_user`, `sbapi_http_pass`.
 
-
-## Datasets API
-
-The URL to retrieve public Images, including the required user and password.
-In general, it shouldn't be modified: `dsapi_url`, `dsapi_http_user`,
-`dsapi_http_pass`.
-
 ## Datacenter details
 
 - `datacenter_name`: The name for the current data center. It's assigned to the
@@ -185,12 +177,10 @@ configuration values required for the correct operation of the UFDS service:
 For every VM used to run one or more of the `SDC` services, we have a set of
 common variables sharing a common naming pattern:
 
-- `${SERVICE_NAME}_root_pw` 
 - `${SERVICE_NAME}_admin_ips`
 - `${SERVICE_NAME}_domain`
 
-Usually, the value for all the `${SERVICE_NAME}_root_pw` is the same, and the
-value for the `${SERVICE_NAME}_domain` matches the pattern
+Usually, the value for the `${SERVICE_NAME}_domain` matches the pattern
 `service_name.datacenter_name.dns_domain` and could be easily figured out from
 these values.
 
